packages/eas-cli/src/build/build.ts

```diff
@@ -202,8 +202,16 @@ export function handleBuildRequestError(error: any, platform: Platform): never {
       `You have already reached the maximum number of pending ${requestedPlatformDisplayNames[platform]} builds for your account. Try again later.`
     );
   } else if (error?.graphQLErrors) {
+    const requestIdLine = error?.graphQLErrors?.[0]?.extensions?.requestId
```

Comment:
This should be handled globally for all graphql errors not only for build request